lib/ecto/datetime.ex

```diff
@@ -167,6 +167,9 @@ defmodule Ecto.Date do
 
   @doc """
   Returns an `Ecto.Date` in local time.
+
+  WARNING: Using the local time of the server will often lead to
+  intermittent bugs. Please use the `utc/0` function instead.
```

Comment:
Yeah, I could expand on it. Maybe I'll do another PR sometime. I will have to phrase it in a good way without it becoming a huge wall of text.